$MirOS: ports/graphics/cairo/patches/patch-src_cairo-type1-subset_c,v 1.2 2008/12/07 17:09:28 bsiegert Exp $
--- src/cairo-type1-subset.c.orig	2009-02-13 11:08:39.000000000 +0100
+++ src/cairo-type1-subset.c	2009-07-17 08:41:29.000000000 +0200
@@ -48,6 +48,8 @@
 #include "cairo-scaled-font-subsets-private.h"
 #include "cairo-output-stream-private.h"
 
+#include <ctype.h>
+
 /* XXX: Eventually, we need to handle other font backends */
 #if CAIRO_HAS_FT_FONT
 
